### PR TITLE
Fix matchmaking + reconnect

### PR DESCRIPTION
**Pull request by Pixailz**
- fixed matchmaking + reconnect
- add debug menu to display nerdy info by pressing F8

### CODE REVIEW

File: src/angularjs/src/app/components/game/started/started.component.ts [hunk 1]
```diff
@@ -282,6 +301,23 @@ export class GameStartedComponent implements OnInit {
...
+				{
+					this.update_sec.text = `${(this.nb_state_change * 1000) / 500}`
+					this.update_ms.text = `${this.serverReceivedTime - this.previousServerReceivedTime}`
+					this.nb_state_change = 0;
+				}
+			},
+		});
+		this.game.add(counter);
+		counter.start();
+	}
```
Comment (clinche) [inline src/angularjs/src/app/components/game/started/started.component.ts:319]: this counter ***intergalaticly destroys*** the game perf on my pc

**Review by clinche — COMMENTED** [review]
reconnect and matchmaking works perfectly even in edge cases (click join, reload, reconnect, and have a second player join), noice!
but the debug menu makes playing the game awful :( is it anticipated to stay in game? if so, i'll try to optimize that, if not we can safely remove it

File: src/angularjs/src/app/components/game/started/started.component.ts (hunk 1 above)
Comment (Pixailz) [reply src/angularjs/src/app/components/game/started/started.component.ts:319]: Nope this was just to display some info, you can safely obliterate it or optimise it if you want :)

**Comment by Pixailz** [comment]
i have commented all the part of the debugmenu, pointing those by the comment `// TODO: DEBUG MENU`

**Review by RuBeeN-cmd — APPROVED** [review]
Noice

**Review by RuBeeN-cmd — APPROVED** [review]
Nicee

**Pull request merged**